### PR TITLE
src/js/common/point.js: remove file

### PR DESCRIPTION
**Pull request by dilyanpalauzov**

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

File: src/js/common/autoScroll.js [hunk 1]
```diff
@@ -162,7 +161,11 @@ AutoScroll.prototype._onMouseDown = function(mouseDownEvent) {
  */
 AutoScroll.prototype._onMouseMove = function(mouseEvent) {
     var edge = this._getEdgePositions(this.container.getBoundingClientRect()),
-        pos = Point.n(domevent.getMousePosition(mouseEvent));
+        position = domevent.getMousePosition(mouseEvent),
```
Comment (jungeun-cho) [inline src/js/common/autoScroll.js:164]: How about changing the name to the `mousePosition` or `point`?

**Comment by dongsik-yoo** [comment]
@dilyanpalauzov I know you contribute many times and I thank you so much. But where is the ISSUE TEMPLATE? I don't believe just issue title is enough to understand what is the problem, why you think so and what you do. I believe a formal template can be helpful to understand for community developers. It's the ISSUE TEMPLATE. That's why I'm telling you.

**Comment by dilyanpalauzov** [comment]
The purpose of this change should be obvious after looking at it: remove unused code from the js-bundle, that is sent to browsers.

I do not see what purpose does the issue template serve, apart from taking time for being filled.

**Comment by jungeun-cho** [comment]
@dilyanpalauzov 
Please respect our issue template form.

**Pull request closed without merging**